x/zoneconcierge/client/cli/query.go

```diff
@@ -24,5 +23,28 @@ func GetQueryCmd(queryRoute string) *cobra.Command {
 		RunE:                       client.ValidateCmd,
 	}
 
+	cmd.AddCommand(CmdChainsInfo())
+	return cmd
+}
+
+func CmdChainsInfo() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "chains-info <chain-ids>",
+		Short: "retrieve the chain info of given chain ids",
+		Args:  cobra.RangeArgs(1, 5),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx := client.GetClientContextFromCmd(cmd)
+			queryClient := types.NewQueryClient(clientCtx)
+			req := types.QueryChainsInfoRequest{ChainIds: args}
+			resp, err := queryClient.ChainsInfo(context.Background(), &req)
```

Comment:
can we use 
```suggestion
			resp, err := queryClient.ChainsInfo(cmd.Context(), &req)
```
here?